gnovm/pkg/gnolang/nodes.go

```diff
@@ -2120,11 +2120,23 @@ func (x *PackageNode) SetBody(b Body) {
 // see tests/selector_test.go.
 type ValuePath struct {
 	Type  VPType // see VPType* consts.
-	Depth uint8  // see doc for ValuePath.
+	depth uint8  // see doc for ValuePath.
 	Index uint16 // index of value, field, or method.
 	Name  Name   // name of value, field, or method.
 }
```

Comment:
I’m not sure why all the fields are  all exported (is this generated from gno.proto?). Could this potentially pose a security issue? cc: @thehowl

Reply:
right missed this one. Maybe we should have a test for that 

Done here: https://github.com/gnolang/gno/pull/3753/commits/3615cf609f248b8eebc2f6e9795a138afae4eb45